main.nf

```diff
@@ -204,6 +204,20 @@ workflow {
             params.gridss_reference_fasta,
             gridss_reference_files
             )
+        gridss_preprocess_dir = preprocess_BAM_GRIDSS.out.gridss_preprocess
+            .collect()
```

Comment:
**comment**: The collect here shouldn't be necessary since there's a `.flatten` immediately after it